### DISCUSSION TITLE
Onboarding Assistant oddness

### DISCUSSION THREAD

**Discussion by jimkohl** [LefterisXris/CodeTour#86] [Ideas]
If I add a step to the base Onboarding Assistant - it creates another Onboarding Assistant instance, totally unexpected.
So then I have to delete the old one.  Ulitmately I try to delete them both but one won't go away.  I found that if I disable the Onboarding Assistant it finally goes away, but it should be easier to just delete it!   While its a great base tour, its not the kind of tour I want to give and ends up being clever unless I disable it.  Just thinking of consumers of this eventually.

**Comment by LefterisXris** [comment]
`Onboarding Assistant` is created in memory and it is hardcoded. It is not meant to be `edited` or `extended`. I am curious, how did you manage to reproduce it?

Add new  Step is disabled on this default Tour:
![image](https://github.com/LefterisXris/CodeTour/assets/22836249/e22c5c18-8a59-46ae-b698-b0d3025d7288)

As well as editing steps:
![image](https://github.com/LefterisXris/CodeTour/assets/22836249/e72c9f25-ca06-4f69-816d-9c5a9365b26d)

In any case, the `Onboarding Assistant` can be easily disabled just by right click on it. I don't think it is hard for anyone to remove it, and I strongly recommend that everyone have a look at it and then decide whether to remove it or not.

**Reply by jimkohl** [reply]
I think I just initially attempted to delete it.  When I couldn't do that from the UI - I did it at the file system - and didn't have great results.

**Reply by LefterisXris** [reply]
The tour file for Virtual Assistant does not exist in your file system. One fix that I did in the latest version, it to exclude `Virtual Onboarding Assistant` from the dropdown selection of the active Tour, when a user wants to add a new step. 

Check the gif file if you want.
![ActiveTourSelection](https://github.com/LefterisXris/CodeTour/assets/22836249/8074579d-4626-43db-9f69-a9999265aa8b)

If that's not your case, please let me know.

**Reply by jimkohl** [reply]
seems like it is good now
